### DISCUSSION TITLE
Unable to fetch letsencrypt: "context deadline exceded"

### DISCUSSION THREAD

**Discussion by fcmircea** [nextcloud/all-in-one#2673] [Questions] [answered]
### Steps to reproduce
1. Install AIO using docker and script
2. Verify the instance is running on :8080
3. Access the SSL interface on: 8443

### Expected behavior 
The AIO instance should obtain the letsencrypt certificate, as it passes the challenge
### Actual behavior 
The AIO returns a "`context deadline exceeded`" error.

Seems like the 10s timeout is too tight, should be adjusted to something more appropriate.

`"max_duration":2592000`

### Host OS 
Ubuntu 22.04 LTS

#### Nextcloud AIO version 
 Nextcloud AIO v5.2.2
#### Other valuable info 
`{"level":"error","ts":1685957827.724578,"logger":"tls.obtain","msg":"could not get certificate from issuer","identifier":"hul9002.go.ro","issuer":"acme-v02.api.letsencrypt.org-directory","error":"[hul9002.go.ro] solving challenges: [hul9002.go.ro] checking authorization status: attempt 1: https://acme-staging-v02.api.letsencrypt.org/acme/authz-v3/6768430494: performing request: Post \"https://acme-staging-v02.api.letsencrypt.org/acme/authz-v3/6768430494\": http2: timeout awaiting response headers (Client.Timeout exceeded while awaiting headers) (order=https://acme-staging-v02.api.letsencrypt.org/acme/order/105329574/9097343464) (ca=https://acme-staging-v02.api.letsencrypt.org/directory)"}`

**Comment by szaimen** [comment]
Hi, after googling your issue I found these two threads:
- https://caddy.community/t/matrix-synapse-getting-no-well-known-error-issues-with-federation/18800/18
- https://caddy.community/t/unable-to-reverse-proxy-for-minio/19472/3

**Comment by fcmircea** [comment]
Hello @szaimen, thank you for the quick turn around.

I'm not at all versed in this topic, but I did restart the router and everything and I'm still facing the issue.

What items from the two replies should I try next or look at to figure it out.

When I open the AIO instance on 8080, I get the `Nextcloud AIO stands for Nextcloud All-in-One and provides easy deployment and maintenance with most features included in this one Nextcloud instance.` screen which allows some kind of domain registration and backup functionality, which I assume it is expected.

the :8443, I still get a timeout

I did forward all the required ports and even placed my server temporarily in the dmz.

**Comment by szaimen** [comment, marked as answer]
Yes, port 8080 is fine to use.
